README.md

```diff
@@ -45,6 +49,17 @@ Or, use it directly via unpkg:
 </script>
 ```
 
+### Using external script reference
+
+```html
+<script src="https://unpkg.com/idb@5.0.4/build/iife/index-min.js"></script>
```

Comment:
```suggestion
<script src="https://unpkg.com/idb@5/build/iife/index-min.js"></script>
```

I'm changing the above to 5 so it doesn't need updated with every release. It mean there's a redirect in practice, but this isn't the main recommended way to use the library anyway.

Reply:
Oh, seems like I can't commit the change above. Can you accept it? Then I'll merge this.

Reply:
I resolved the conversation. Thanks